.gitignore

```diff
@@ -11,6 +11,8 @@
 /stemwords
 /dist
 /java/org/tartarus/snowball/ext/
+/javascript/*-stemmer.js
+!/javascript/base-stemmer.js
```

Comment:
The generated sources are actually put in `js_out/` (and `NODE_PATH` is set so both directories are searched) so these ignore rules shouldn't be needed.